packages/react-topology/src/pipelines/components/nodes/TaskNode.tsx

```diff
@@ -45,6 +45,7 @@ export type TaskNodeProps = {
   badgeTextColor?: string;
   badgeBorderColor?: string;
   badgeClassName?: string;
+  badgeTooltip?: React.ReactNode;
```

Comment:
Is `badgePopoverProps` being used anywhere? Should it be included in the logic on line282 and spread to the Popover as well as `badgePopoverParams`?

It is likely worth noting in a comment that you cannot use both badgeTooltip and badgePopoverParams at the same time, only the tooltip would be rendered in that case.

Reply:
`badgePopoverProps` is a leftover, it is now `badgePopoverParams`

Reply:
added comments